packages/navitem/src/react/index.js

```diff
@@ -2,22 +2,14 @@ import React from 'react'
 import PropTypes from 'prop-types'
 
 import { alignments } from '../vars/index.js'
-import { Bar, Button, Container } from './common.js'
 import { HorzLayout } from './horz.js'
 import { VertLayout } from './vert.js'
 
 const NavItem = React.forwardRef((props, forwardedRef) => {
-  return (
-    <Container>
-      <Button {...props} ref={forwardedRef}>
-        {props.alignment === 'vertical' ? (
-          <VertLayout {...props} />
-        ) : (
-          <HorzLayout {...props} />
-        )}
-      </Button>
-      <Bar {...props} />
-    </Container>
+  return props.alignment === 'vertical' ? (
```

Comment:
Now Container is not common, but vert/horz-specific.  Moved this whole "layout" into the *Layout functions.